static/_redirects

```diff
@@ -6,6 +6,7 @@
 
 /concepts/containers/container-lifecycle-hooks/     /docs/concepts/containers/container-lifecycle-hooks/ 301
 /docs/     /docs/home/ 301!
+/bn/docs      /bn/docs/home/ 301!
```

Comment:
```suggestion
/bn/docs/      /bn/docs/home/ 301!
```

Reply:
Done in https://github.com/kubernetes/website/pull/46297